src/components/Cashout/Components/Initial.view.tsx

```diff
@@ -10,37 +10,34 @@ import * as context from '@/context'
 import Loading from '@/components/Global/Loading'
 import { useBalance } from '@/hooks/useBalance'
 import { useAuth } from '@/context/authContext'
-import { useForm } from 'react-hook-form'
-import { useWalletType } from '@/hooks/useWalletType'
 import { useCreateLink } from '@/components/Create/useCreateLink'
-import { GlobalLoginComponent } from '@/components/Global/LoginComponent'
 import { Icon as ChakraIcon } from '@chakra-ui/react'
 import * as assets from '@/assets'
 import * as utils from '@/utils'
 import { FAQComponent } from './Faq.comp'
 import { RecipientInfoComponent } from './RecipientInfo.comp'
 import { motion, AnimatePresence } from 'framer-motion'
 import Icon from '@/components/Global/Icon'
+import { twMerge } from 'tailwind-merge'
 
 export const InitialCashoutView = ({
     onNext,
     tokenValue,
     usdValue,
     setUsdValue,
     setTokenValue,
-    setRecipient,
-    recipient,
```

Comment:
thought: both `setRecipient` and `recipient` are now not-used in `InitialCashoutView` now. We should prob clean it up